src/qcodes/instrument/visa.py

```diff
@@ -45,6 +47,51 @@ def _close_visa_handle(
         # the resource is already closed
         pass
 
+
+class VisaInstrumentKWArgs(TypedDict):
+    """
```

Comment:
and we can't or we should'nt inherit from InstrumentBaseKWArgs?

Reply:
Minimal example of this can be seen here https://github.com/jenshnielsen/debugproject/tree/error_on_typed_dict_inherit